static/app/components/sentryAppComponentIcon.spec.tsx

```diff
@@ -0,0 +1,24 @@
+import {SentryAppComponentFixture} from 'sentry-fixture/sentryAppComponent';
+
+import {getSentryAppComponentIsDisabled} from 'sentry/components/sentryAppComponentIcon';
+
+describe('SentryAppComponentIcon', function () {
+  it('returns false if the error is a non empty string', () => {
+    const component = SentryAppComponentFixture();
+    component.error = 'RIP couldnt connect to sentry :C';
+    expect(getSentryAppComponentIsDisabled(component)).toBe(false);
+  });
+
+  it('returns true if the error is an empty string', () => {
+    const component = SentryAppComponentFixture();
+    component.error = '';
+    expect(getSentryAppComponentIsDisabled(component)).toBe(true);
+  });
+
+  // TODO: Delete after new errors are deployed
```

Comment:
> new errors are deployed

What's this process gonna look like?

Reply:
Once #81167 is deployed to all regions we can change the `isDisabled` check to just use the logic of when `error` is a string, instead of having to check if `error` is a boolean or string. So in that sense, we can delete all the tests we in this PR and `getSentryAppComponentIsDisabled` .